docs/Available-Symbols-Generators.md

```diff
@@ -458,3 +490,69 @@ In this case, if the user enters the value `123` as the value of the parameter `
 
 ### Related
 [`Implementation class`](https://github.com/dotnet/templating/blob/master/src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Macros/SwitchMacro.cs)
+
+## Join
+
+Concatenates multiple symbols or constants with defined separator into new symbol.
+
+#### Parameters
+| Name     |Data Type| Description   |    
+|----------|---------|---------------|    
+|symbols   |array    |all values to concatenate|
+|separator |string   |any string, can also be empty|
```

Comment:
The purpose of separator is not clear here. Consider changing to similar to "the value used as the separator between the values to be concatenated".
If some properties are optional, it would be good to add the separate column to indicate it.